worker/pkg/workflows/datasync/activities/activities.go

```diff
@@ -727,6 +728,11 @@ func computeMutationFunction(col *mgmtv1alpha1.JobMapping, colInfo *dbschemas_ut
 		maxLen = *colInfo.CharacterMaximumLength
 	}
 
+	// Use the global RNG to seed the transformer RNGs
+	seed := rand.Int()
+
+	seedValue := rand.New(rand.NewSource(int64(seed)))
```

Comment:
can we just leave this out for now and let the transformer internals generate this in the init method?

Reply:
Yeah we can do that, then we just won't pass it in. Updated.